File: javascript/Mobile.js
// javascript/Mobile.js
import { ShippableProduct } from './ShippableProduct.js';

export class Mobile extends ShippableProduct {
    constructor(name, price, quantity, weight) {
        super(name, price, quantity, weight);
    }

    isExpired() {
        return false;
    }
}

File: csharp/Mobile.cs
// csharp/Mobile.cs
namespace ECommerce
{
    public class Mobile : ShippableProduct
    {
        public Mobile(string name, double price, int quantity, double weight)
            : base(name, price, quantity, weight)
        {
        }

        public override bool IsExpired()
        {
            return false;
        }
    }
}
